x-pack/solutions/observability/plugins/observability/public/pages/alert_details/hooks/use_related_dashboards.test.ts

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { renderHook } from '@testing-library/react';
+import { useRelatedDashboards } from './use_related_dashboards';
+import { kibanaStartMock } from '../../../utils/kibana_react.mock';
+
+const mockUseKibanaReturnValue = kibanaStartMock.startContract();
+let capturedQueryFn: (() => Promise<any>) | undefined;
+
+// Test constants
+const TEST_ALERT_ID = 'test-alert-id';
+const API_ENDPOINT = '/internal/observability/alerts/related_dashboards';
+
+const TEST_DASHBOARD_1 = {
+  id: 'dashboard-1',
+  title: 'Dashboard 1',
+  description: 'This is dashboard 1',
+};
+
+const TEST_DASHBOARD_2 = {
+  id: 'dashboard-2',
+  title: 'Dashboard 2',
+  description: 'This is dashboard 2',
+};
+
+jest.mock('@kbn/triggers-actions-ui-plugin/public', () => ({
+  __esModule: true,
+  useKibana: jest.fn(() => mockUseKibanaReturnValue),
+}));
+
+const mockUseQuery = jest.fn();
+jest.mock('@tanstack/react-query', () => ({
+  useQuery: (params: { queryKey: string[]; queryFn: () => Promise<any> }) => mockUseQuery(params),
+}));
+
+describe('useSuggestedDashboards', () => {
```

Comment:
The file is called `use_related_dashboards` which refers to both suggested and linked dashboards. Maybe you can add one test for linkedDashboards as well?